docs/framework/guides/package-generator/javascript-package.md

```diff
@@ -231,9 +199,13 @@ Examples:
 npm run translations:upload -- --organization [ORGANIZATION] --project [PROJECT]
 ```
 
-### `prepare` and `prepublishOnly`
+### `prepare`
```

Comment:
Why the `prepublishOnly` script is removed from here?

Reply:
We found out that `prepare` triggers both, at package creation, and at publishing, while `prepublishOnly` only triggers at the latter. Since they were both calling `dll:build`, it resulted in redundant double call of that script before publishing, and thus the script was removed.